src/components/shared/forms/styles/pill.scss

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/themes';
+@import 'src/styles/mixins';
+
+$pillHeight: 30px;
+$pillMinWidth: 127px;
+$pillIconSize: 16px;
+$pillIconWrapperSize: 30px;
+
+.pill {
+  display: flex;
+  flex-shrink: 0;
+  justify-content: space-between;
+  align-items: center;
+  text-align: left;
+  white-space: nowrap;
+  cursor: pointer;
+  font-family: inherit;
+  @include rem-fallback(margin, 7px, 7px, 0px, 0px);
+  @include rem-fallback(padding, 0px, $pillHeight/4);
+  @include rem-fallback(min-width, $pillMinWidth);
+
+  .pill-icon {
+    position: relative;
+    @include rem-fallback(height, $pillHeight);
+    @include rem-fallback(width, $pillIconWrapperSize);
+    @include rem-fallback(margin-right, -8px);
+
+    svg { @include square-px-rem($pillIconSize); }
+
+    + .pill-text { @include rem-fallback(padding-left, 5px); }
```

Comment:
This needs padding on the right as well, right? Otherwise, the text may be butted up against the color change.
![image](https://user-images.githubusercontent.com/16829221/47323609-5abe2900-d611-11e8-9be9-11aced987f2e.png)

Reply:
fixed